dev-packages/cli/README.md

```diff
@@ -56,11 +56,15 @@ If you set `next` in your theia config, then Theia will prefer `next` over `late
 
 To build once:
 
-    theia build
+    NODE_ENV=development theia build
```

Comment:
It should be `theia build` by default, we could add  below that to build in dev mode `--mode development` should be added.

With webpack 4 we should use `--mode` instead of `NODE_ENV`.